pkg/virtctl/guestfs/guestfs.go

```diff
@@ -59,6 +59,12 @@ type guestfsCommand struct {
 	pullPolicy   string
 }
 
+// Following variables allow overriding the default functions (useful for unit testing)
+var CreateClientFunc = createClient
```

Comment:
If it is not overridden anywhere, you might drop this line?

Reply:
in tests it is few times
i.e
https://github.com/kubevirt/kubevirt/pull/13133/files#diff-fd7210d9ec106ebefbdaa02059d8b18ef4da7ab64bda97b5b33be96e28034f2bR120

```
pkg/virtctl/guestfs/guestfs_test.go:                    guestfs.CreateClientFunc = fakeCreateClientPVC
pkg/virtctl/guestfs/guestfs_test.go:                    guestfs.CreateClientFunc = fakeCreateClientPVCinUse
pkg/virtctl/guestfs/guestfs_test.go:                    guestfs.CreateClientFunc = fakeCreateClient
pkg/virtctl/guestfs/guestfs_test.go:                    guestfs.CreateClientFunc = fakeCreateClientPVC
pkg/virtctl/guestfs/guestfs_test.go:                    guestfs.CreateClientFunc = fakeCreateClientPVC
```

Reply:
Is a cleanup missing then? `createClient` is not exported.

Reply:
not sure how it can affect, as it was already the state before, `SetDefaulClient` was not called on AfterEach
but will add it